src/cmd/linuxkit/moby/config.go

```diff
@@ -590,6 +591,7 @@ func assignRuntime(v1, v2 *Runtime) Runtime {
 	runtimeMkdir := assignStrings(v1.Mkdir, v2.Mkdir)
 	runtimeInterfaces := assignRuntimeInterfaceArray(v1.Interfaces, v2.Interfaces)
 	runtimeNamespace := assignString(v1.Namespace, v2.Namespace)
+	runtimeEnvFromFile := assignMaps(v1.EnvFromFile, v2.EnvFromFile)
```

Comment:
I didn't realise I had to do this had to spend some time debugging why my change didn't work. Perhaps we need to comment structs with references to other copies of the same struct definition and this function also.

Reply:
It is kind of ugly and verbose, maybe we could replace all this with `reflect`?

Reply:
Not too sure. Perhaps much of these concerns can be solved by comments that someone updating either of the structs will read. It's okay that this is not pretty, just would be nice to know all parts of the code that need updating. I don't know if this is just the case with `Runtime`, or there are other structs that are handled in the same way like this.

Reply:
I think this is the only case, could probably be fixed by exporting the types from a library instead I guess (I can't remember why it is like this and I don't think it has to be).